common/presenters/message-banner/move-to-message-banner-component.js

```diff
@@ -24,34 +24,32 @@ function moveToMessageBannerComponent({ move = {}, moveUrl, canAccess } = {}) {
     return assessmentToStartBanner({ baseUrl, canAccess, context })
   }
 
-  if (assessment?.status === 'confirmed' && assessment.handover_occurred_at) {
-    return assessmentToHandedOverBanner({
-      assessment,
-      baseUrl,
-      canAccess,
-      context,
-    })
-  }
-
   if (assessment?.status === 'confirmed') {
-    return assessmentToConfirmedBanner({
-      assessment,
-      baseUrl,
-      canAccess,
-      context,
-    })
-  }
-
-  if (assessment?.status !== 'confirmed') {
-    return assessmentToUnconfirmedBanner({
-      assessment,
-      baseUrl,
-      canAccess,
-      context,
-    })
+    if (assessment.handover_occurred_at) {
+      return assessmentToHandedOverBanner({
+        assessment,
+        baseUrl,
+        canAccess,
+        context,
+      })
+    }
+
+    if (assessmentType === 'youth_risk_assessment') {
+      return assessmentToConfirmedBanner({
+        assessment,
+        baseUrl,
+        canAccess,
+        context,
+      })
+    }
   }
 
-  return undefined
```

Comment:
Won't removing this mean the unconfirmed banner is always displayed, even when a PER hasn't been started yet?

Reply:
I don't think I've changed any behaviour by removing this, previously we were checking against one conditional, then the reverse of that conditional, so I don't think this would've been hittable?

Reply:
Yeah it's odd we didn't have a test for this and I'm trying to rack my brain about why. I think it was if there was no status, which in theory is everything about line 36 so this looks fine.